incubator-tools/cross_project_docai_processor_calling/cross project.ipynb

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
<p>I don't think this notebook is needed. It's already in the documentation here <a href="https://cloud.google.com/document-ai/docs/cross-project-setup" target="_blank">https://cloud.google.com/document-ai/docs/cross-project-setup</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/749/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done